cfgov/unprocessed/css/skip-nav.less

```diff
@@ -20,8 +20,13 @@
 }
 
 #skip-nav:focus {
-    .btn;
-    .btn__super;
+    .a-btn;
```

Comment:
This one can't be since it's only added on `focus`, so it can't be there all the time.